201-logic-app-as2-send-receive/README.md

```diff
@@ -0,0 +1,36 @@
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-logic-app-as2-send-receive%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+This template demonstrates AS2 Send Receive using Logic Apps. It creates integration accounts for two partners (Contoso and Fabrikam) and adds Partners and Agreements into them. It creates logic apps between Fabrikam Sales and Contoso which demonstrate Sync AS2 Send Receive. It also creates logic apps between Fabrikam Finance and Contoso which demonstrate ASync AS2 Send Receive.
+`Tags: AS2, Logic Apps, Integration Account, Enterprise Integration`
+
+## Deployment steps
+
+You can click the "deploy to Azure" button at the beginning of this document or follow the instructions for command line deployment using the scripts in the root of this repo.
+
+## Usage
+
+Once the deployment is completed, you can perform below steps to test your Logic Apps:
+- Open the resource group blade in Azure Portal where you deployed all resources.
+![Image of Azure resources](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/201-logic-app-as2-send-receive/images/azure-resources.png "Azure resources"))
+- The FabrikamSales-AS2Send and Contoso-Receive logic apps demonstrate sync send receive scenario. Open the FabrikamSales-AS2Send Logic App blade and click on Run Trigger to run it.
+![Image of FabrikamSales-AS2Send Logic App](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/201-logic-app-as2-send-receive/images/fabrikamsales-as2send.png "Run FabrikamSales-AS2Send Logic App"))
+-- You can look into the run history and input/output for each action for these logic apps.
+![Image of Contoso-AS2Receive run history](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/201-logic-app-as2-send-receive/images/contoso-as2receive-runhistory.png "Contoso-AS2Receive run history"))
+- The FabrikamFinance-AS2Send and Contoso-Receive logic apps demonstrate async send receive scenario. Open the FabrikamFinance-AS2Send Logic App blade and click on Run Trigger to run it.
+-- The async MDN is received by the FabrikamFinance-AS2ReceiveMDN logic app.
+![Image of FabrikamFinance-AS2ReceiveMDN run history](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/201-logic-app-as2-send-receive/images/fabrikamfinance-as2receivemdn-runhistory.png "FabrikamFinance-AS2ReceiveMDN run history"))
+-- Again, you can look into the run history and input/output for each action for these logic apps.
+
+## Notes
+
+Learn more about: Azure Logic Apps
+* **Azure Logic Apps** - https://azure.microsoft.com/en-us/documentation/articles/app-service-logic-what-are-logic-apps/
```

Comment:
remove en-us. Leaving this culture code out ensures that the viewer gets docs in their own language